lib/unparser/writer/send.rb

```diff
@@ -75,20 +75,7 @@ def arguments
       end
 
       def emit_normal_arguments
-        parentheses do
-          arguments.each_with_index do |node, index|
-            write(', ') unless index.zero?
-            emit_argument(node, index.succ.equal?(arguments.length))
-          end
-        end
-      end
-
-      def emit_argument(argument, last)
-        if n_hash?(argument) && last
```

Comment:
I *really* like this special case to be gone, its now a first class signal via the new `kwargs` node.